roles/st2/tasks/config_auth.yml

```diff
@@ -1,11 +1,29 @@
-- name: Install auth pre-reqs
+- name: Install auth pre-reqs (Debian)
   become: yes
   apt:
     name: "{{ item }}"
     state: present
   with_items:
     - python-passlib
     - apache2-utils
+  when: ansible_os_family == 'Debian'
+
+- name: Install epel-release repo (RedHat)
+  become: yes
+  yum:
+    name: epel-release
```

Comment:
Will this work on real `RHEL6`?
You can test in AWS https://aws.amazon.com/marketplace/pp/B00CFQWLS6